app/jobs/sync_track_update_job.rb

```diff
@@ -0,0 +1,13 @@
+class SyncTrackUpdateJob < ApplicationJob
+  after_perform do |job|
+    track_update = TrackUpdate.find(job.arguments.first)
+
+    track_update.update!(synced_at: Time.current)
+  end
+
+  def perform(track_update_id)
+    track_update = TrackUpdate.find(track_update_id)
+
+    Git::SyncsTracks.sync([track_update.track])
```

Comment:
I wonder if we can just remove the `SyncsTracks` class now that we're not looping through everything? What do you think. There's obviously that hacky bit of code about `hello-world` to address, but that can go inside `SyncsTrack` I think.

Reply:
In the previous PR, you mentioned a nightly task that pulls tracks just in case the webhooks don't work. I think having the `SyncsTracks` service helps there since we can only call `SyncsTracks.sync(Track.all)`.

Reply:
Good point. I see